packages/stores/src/schema-store/actions/execute-operation.ts

```diff
@@ -18,6 +32,50 @@ import type {
   ExecutionResponse,
   SchemaStoreActions,
 } from "../schema-store.types";
+import { Fragment } from "react";
+
+enum Directive {
+  Defer = "defer",
+}
+
+const usingDefer = (set: SelectionSetNode | undefined): boolean =>
+  set?.selections.some((selection) => {
+    const fragmentUse =
+      selection.kind == Kind.INLINE_FRAGMENT ||
+      selection.kind == Kind.FRAGMENT_SPREAD;
+    const usingDeferOnCurrentSelection =
+      fragmentUse &&
+      (selection.directives?.some(
+        (node) => node.name.value === Directive.Defer
+      ) ??
+        false);
+    const hasSelectionSet = "selectionSet" in selection;
+    return (
+      usingDeferOnCurrentSelection ||
+      (hasSelectionSet && usingDefer(selection.selectionSet))
+    );
+  }) ?? false;
+
+const mergeResults = (
+  result: ExecutionResult<Record<string, unknown>, unknown>,
+  lastResult?: ExecutionResult<Record<string, unknown>, unknown>
+) => {
+  // bit of weird typing here, result should probably be returned as ExecutionResult | ExecutionPatchResult from graphql-sse
+  // but that isn't the case
+  if (!("path" in result) || lastResult == null) {
+    return result;
+  }
+
+  const path = result.path as string[];
+
+  const combined = update(lastResult, ["data", ...path], (value) =>
+    merge(value, result.data)
+  );
+
+  const errors = [...(lastResult?.errors ?? []), ...(combined.errors ?? [])];
```

Comment:
lastResult.errors & combined.errors are the same at this point, no?  Think you want:

```suggestion
  const errors = [...(lastResult?.errors ?? []), ...(result ?? [])];
```

Reply:
Yes thanks, probably a good time for a cup of coffee